util/patch/patch_test.go

```diff
@@ -20,23 +20,25 @@ import (
 	"context"
 	"testing"
 
+	"github.com/davecgh/go-spew/spew"
```

Comment:
I would expect this to be grouped with the imports below.

Reply:
This should have been removed, thanks!